dll/win32/shell32/shlfileop.cpp

```diff
@@ -1742,6 +1742,9 @@ static void move_to_dir(FILE_OPERATION *op, const FILE_ENTRY *feFrom, const FILE
 
     PathCombineW(szDestPath, feTo->szFullPath, feFrom->szFilename);
 
+    if (feFrom->attributes == INVALID_FILE_ATTRIBUTES)
+        return;
+
```

Comment:
```suggestion
    if (feFrom->attributes == INVALID_FILE_ATTRIBUTES)
        return;

    PathCombineW(szDestPath, feTo->szFullPath, feFrom->szFilename);

```
And I would assume the new check can even be moved to before directory creation, can't it?

Reply:
Fixed. Thanks.